lab and lab.bat scripts are corrupt in recent dev builds

### Environment
What version of MekLab does your issue apply to?  current master branch 
What operating system are you using?  Windows, Mac, or Linux? Linux
What java version are you using?  If unsure it will state your java version at the top of the megameklablog.txt file.
1.8.0

### Description
The lab scripts (the Linux shell script and Windows batch script) are both corrupt in my build.  The following gets written in CLASSPATH:
```
CLASSPATH=$APP_HOME/MegaMekLab.property(class java.lang.String, fixed(class java.lang.String, jar)):$APP_HOME/lib/svgSalamander.jar:$APP_HOME/lib/megamek-0.47.5-SNAPSHOT.jar:$APP_HOME/lib/log4j-1.2.17.jar:$APP_HOME/lib/batik-r....
```

The script is syntactically incorrect because of the parentheses.

Additionally, when replacing that with the proper MegaMekLab.jar, trying to run the script results in:
```
 ./lab 
Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.NoClassDefFoundError: megamek/common/logging/MMLogger
	at java.lang.Class.getDeclaredMethods0(Native Method)
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
	at java.lang.Class.privateGetMethodRecursive(Class.java:3048)
	at java.lang.Class.getMethod0(Class.java:3018)
	at java.lang.Class.getMethod(Class.java:1784)
	at sun.launcher.LauncherHelper.validateMainClass(LauncherHelper.java:544)
	at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:526)
Caused by: java.lang.ClassNotFoundException: megamek.common.logging.MMLogger
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
	... 7 more
```

--- a/build.gradle
+++ b/build.gradle
@@ -121,8 +121,8 @@ task compileMM (type: GradleBuild) {
 
 task mmJar (type: Jar, dependsOn: compileMM) {
     archiveFileName = "MegaMek.jar"
-    from "${mmDir}/megamek/megamek/build/classes/java/main"
-    from "${mmDir}/megamek/megamek/build/resources/main"
+    from "${mmDir}/megamek/build/classes/java/main"
+    from "${mmDir}/megamek/build/resources/main"
     manifest {
         attributes "Main-Class": mainClassName
         attributes 'Class-Path' : (project.sourceSets.main.runtimeClasspath.files

--- a/build.gradle
+++ b/build.gradle
@@ -130,14 +130,14 @@ task mmJar (type: Jar, dependsOn: compileMM) {
     }
     inputs.dir "${mmDir}/megamek/build/classes/java/main"
     inputs.dir "${mmDir}/megamek/build/resources/main"
-    outputs.file "${buildDir}/libs/${archiveFileName}"
+    outputs.file "${buildDir}/libs/${archiveFileName.get()}"
 }
 
 jar {
-    archiveFileName = "${rootProject.name}.${archiveExtension}"
+    archiveFileName = "${rootProject.name}.${archiveExtension.get()}"
     manifest {
         attributes "Main-Class": mainClassName
-        attributes 'Class-Path' : "${lib}/${mmJar.archiveFileName} " + (project.sourceSets.main.runtimeClasspath.files
+        attributes 'Class-Path' : "${lib}/${mmJar.archiveFileName.get()} " + (project.sourceSets.main.runtimeClasspath.files
             .findAll { it.name.endsWith(".jar") }.collect { "${lib}/${it.name}" }.join(' '))
     }
 }
@@ -225,7 +225,7 @@ task createOSXBundle (dependsOn: jar) {
                 mainClass: project.mainClassName,
                 stubfile: osxApplicationStub,
                 dir: osxBundleDir,
-                jar: "${jar.destinationDirectory}/${jar.archiveFileName}",
+                jar: "${jar.destinationDirectory.get()}/${jar.archiveFileName.get()}",
                 jvmversion: '1.8+',
                 icon: "${projectDir}/${data}/images/misc/megameklab.icns",
                 useJavaXKey: 'true',

